### ISSUE TITLE
When executing after registering the shortcut key, the path of the current file is appended and entered.

### ISSUE THREAD

**Issue by ski0090** [edonet/vscode-command-runner#46]
When executing after registering the shortcut key, the path of the current file is appended and entered.

keybindings.json

```json

[
    {
        "key": "ctrl+f5",
        "command": "command-runner.run",
        "args": {
            "command": "cargo fmt; cargo clippy --workspace --all-targets --verbose",
        }
    },
    {
        "key": "f5",
        "command": "command-runner.run",
        "args": {
            "command": "cargo fmt; cargo clippy --workspace --all-targets --verbose; cargo run",
        }
    },
    {
        "key": "ctrl+shift+b",
        "command": "command-runner.run",
        "args": {
            "command": "meson build --prefix C:\\deploy --reconfigure; ninja -C build install"
        }
    },
]
```

Ctrl + shift + b
```powershell
meson build --prefix C:\deploy --reconfigure; ninja -C build install "C:\\Users\\Shalom\\AppData\\Roaming\\Code\\User\\keybindings.json"
```

F5
```powershell
 cargo fmt; cargo clippy --workspace --all-targets --verbose; cargo run "C:\\Users\\Shalom\\AppData\\Roaming\\Code\\User\\keybindings.json"
```

**Comment by edonet** [comment]
Thinks for issue, fixed with 0.0.122, enjoy it!

**Comment by ski0090** [comment]
![download](https://user-images.githubusercontent.com/45457350/148025548-120c85ef-7286-4c35-86a5-88b460d80a0c.gif)
